.github/workflows/spelling.yml

```diff
@@ -0,0 +1,19 @@
+name: Spelling
+
+on:
+  pull_request:
+  push:
+
+permissions:
+  contents: read
```

Comment:
Actually the read permission is not needed for public repos.
https://github.com/szepeviktor/byte-level-care/blob/master/.github/workflows/spelling.yml#L13

Reply:
not sure if it is still the case, but there was a moment when if no permissions declared explicitly, some more than "read" was allowed.

explicit > implicit